### DISCUSSION TITLE
I also need an help about generate sandbox reports

### DISCUSSION THREAD

**Discussion by NSH531** [CrowdStrike/falconpy#675] [Q&A] [answered]
how to do this???

this discussion is an appendix to https://github.com/CrowdStrike/falconpy/discussions/672

**Comment by jshcodes** [comment, marked as answer]
Hi @NSH531 -

Reports are generated once you [submit](https://www.falconpy.io/Service-Collections/Falconx-Sandbox.html#submit) the file for analysis.  (~ 60 seconds for Quick Scan, and between 1 to 15 minutes for ML scan.) 

You can [query](https://www.falconpy.io/Service-Collections/Falconx-Sandbox.html#queryreports) for reports using a [FQL filter](https://www.falconpy.io/Usage/Falcon-Query-Language.html), and retrieve specific reports by their unique ID. (Either the [full report](https://www.falconpy.io/Service-Collections/Falconx-Sandbox.html#getreports) or the [summary](https://www.falconpy.io/Service-Collections/Falconx-Sandbox.html#getsummaryreports).)

An example of reviewing the results of a Sandbox analysis can be found in our [Falcon X Single Scan](https://github.com/CrowdStrike/falconpy/tree/main/samples/falconx_sandbox/single_scan/falconx_scan_example.py#L255) sample.

Thank you for the question!
